assemblies/assembly_planning/main.adoc

```diff
@@ -110,7 +110,7 @@ include::{enterprise-dir}/admin-and-config/maven-con.adoc[leveloffset=+2]
 include::{shared-dir}/KIE/Overview/installation-options-ref.adoc[leveloffset=+1]
 
 //Difference between a clustered and a standalone environment. Pros and cons of each. Discuss //high availability. Clustering a Development environment vs Clustering a Production environment
-//Include a descriptions of NFS, Artemis, Elasticsearch, any other cluster-related tools.
+//Include a descriptions of NFS, Artemis, Data Grid, any other cluster-related tools.
```

Comment:
is this supposed to be commented?

Reply:
These are notes for including a fully supported HA summary for 7.5.